packages/json-schemas/tsconfig.json

```
@@ -2,7 +2,47 @@
     "extends": "../../tsconfig",
     "compilerOptions": {
         "outDir": "lib",
-        "rootDir": "."
+        "rootDir": ".",
+        "resolveJsonModule": true
     },
-    "include": ["./src/**/*", "./test/**/*", "./schemas/**/*"]
+    "include": ["./src/**/*", "./test/**/*", "./schemas/**/*"],
```

You can remove the schemas from here.
```suggestion
    "include": ["./src/**/*", "./test/**/*"],
```